src/Main.java

```diff
@@ -1,81 +1,53 @@
+import javax.swing.*;
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
 
-        // task1
+            //task #1
 
-        byte clientOS = 1;
-        if (clientOS == 0) {
-            System.out.println("Установите версию для iOS по ссылке.");
-        } else {
-            System.out.println("Установите версию для Android по ссылке.");
+        int[] sumCost = {30, 25, 29, 32, 33};
+        int sum = 0;
+        for (int element : sumCost) {
+            sum = sum + element;
         }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей.");
 
-        // task2
+            //task #2
 
-        byte clientOS1 = 0;
-        int clientDeviceYear = 2014;
-        if (clientOS1 == 1 && clientDeviceYear < 2015) {
-            System.out.println("Установите облегченную версию приложения для Android по ссылке.");
-        } else if (clientOS1 == 0 && clientDeviceYear < 2015) {
-            System.out.println("Установите облегченную версию приложения для iOS по ссылке.");
-        }
-        if (clientOS1 == 0 && clientDeviceYear >= 2015) {
-            System.out.println("Установите версию для iOS по ссылке.");
-        } else if (clientOS1 == 1 && clientDeviceYear >= 2015) {
-            System.out.println("Установите версию для Android по ссылке.");
+        int[] cost = {30, 25, 29, 32, 33};
+        int minCost = 40;
```

Comment:
возьми первое значение из массива, тогда у тебя логика будет работать при любых диапазонах значений
```suggestion
        int minCost = cost[0];
```